src/SeqCli/Cli/Commands/ApiKey/CreateCommand.cs

```diff
@@ -80,7 +80,7 @@ public CreateCommand(SeqConnectionFactory connectionFactory, SeqCliConfig config
 
         Options.Add(
             "connect-username=",
-            "A username to connect with, useful primarily when setting up the first API key",
+            "A username to connect with, useful primarily when setting up the first API key; servers with an 'Individual' subscription only allow one simultaneous request with this option",
```

Comment:
Ah, I see - each established session closes any others 👍 

Normally this wouldn't be a problematic limitation, since later requests are expected to use an API key generated through this command.